### DISCUSSION TITLE
Build static site

### DISCUSSION THREAD

**Discussion by jgalazm** [executablebooks/meta#947] [Q&A] [answered]
Hi, is it possible to build a static site using the `myst build` command ? 

I see that I can:
- start a local server with `myst start` and render the website in "live" mode
- export to pdf, word, etc, with `myst build` and the exports key in the frontmatter

but I can not find the way to get just the html, js, css and assets files to deploy it with a simple web server.

Is that possible?

**Comment by rowanc1** [comment, marked as answer]
Hi @jgalazm -- an option at this stage is to use Curvenote to [deploy the site](https://curvenote.com/docs/web/tutorial-deploy-local#xwBEM7e27l) (or use a [github action](https://curvenote.com/docs/web/github-action)).
We are tracking the feature for static HTML export of the site here:

- https://github.com/executablebooks/mystjs/issues/188

This is rising high on the priority list, but still being blocked by a few tasks! Hope that helps for now.

**Reply by jgalazm** [reply]
Hi there @rowanc1 , I saw the new deployment option and it is exactly what I wanted. Thanks again!

**Reply by rowanc1** [reply]
Awesome! Glad you found it!!
